src/Integrations/Integrations/WordPress/WordPressIntegration.php

```diff
@@ -42,10 +43,15 @@ public function init()
                 return false;
             }
             $majorVersion = substr($GLOBALS['wp_version'], 0, 1);
-            if ($majorVersion >= 4) {
+            if ($majorVersion >= 6) {
+                $loader = new WordPressIntegrationLoader();
```

Comment:
Didn't you mean WordPressComponent or something?

Reply:
Yes 😅 It was just to take the screenshots that I changed it. Forgot to change it back before committing